CMakeLists.txt

```diff
@@ -2,10 +2,9 @@
 cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
 
 if(POLICY CMP0074)
-  # TODO:
-  # 1. Find*.cmake modules need to be individually verified.
-  # 2. PCLConfig.cmake needs to be changed.
-  cmake_policy(SET CMP0074 OLD)
+  # 1. Remove with 3.12.4.
```

Comment:
What do you mean by this?

Reply:
But if the policy is not set CMake will default to OLD?

Reply:
Let me write a couple of cases to clarify:
- Case 1:
  - Policy: Not Set
  - CMake minimum required version: 3.1
  - CMake version used: 3.1
  - Results: You get the OLD behavior. CMake 3.1 doesn't even know about this policy because it was only created in version 3.12 it he behaves with the OLD behavior.
- Case 2:
  - Policy: Not Set
  - CMake minimum required version: 3.1
  - CMake version used: 3.12
  - Results: You get the OLD behavior by default and a warning saying to explicitly set the bhavior. CMake 3.12 wants you to know that things changed.
- Case 3:
  - Policy: Not Set
  - CMake minimum required version: 3.12
  - CMake version used: 3.12
  - Results: You get the new behavior by default. 
- Case 4:
  - Policy: Explicitly set to NEW
  - CMake minimum required version: 3.1
  - CMake version used: 3.1
  - Results: CMake 3.1 doesn't even know about this policy and ignores it. It still behaves in the OLD behavior.
- Case 5:
  - Policy: Explicitly set to NEW
  - CMake minimum required version: 3.1
  - CMake version used: 3.12
  - Results: You get the NEW behavior. You're in a situation in which you're compliant with CMake versions from the minimum required one all the way to 3.12+
- Case 6:
  - Policy: Explicitly set to NEW
  - CMake minimum required version: 3.12
  - CMake version used: 3.12
  - Results: You get the new behavior by default but now you're explicitly setting it although it isn't required.

Reply:
Thanks for the detailed list, I wasn't aware that case 3 behaves this way. Although it's the most logical thing to do. And from CMake docs:

> The cmake_minimum_required() command does more than report an error if a too-old version of CMake is used to build a project. It also sets all policies introduced in that CMake version or earlier to NEW behavior.